### ISSUE TITLE
Zabbix agent 6.4.2 enters runaway in 15-3.1.0 release

### ISSUE THREAD

**Issue by radarsymphony** [tiredofit/docker-postgres#6]
Hi tiredofit,

When I pull your latest postgres image and run it with your [example compose.yml](https://github.com/tiredofit/docker-postgres/blob/92ec9df6d387b261e6d596e2f438eb0ffc9b2636/examples/compose.yml#L1), I see the zabbix agent isn't starting up as expected. Here are the docker-compose logs:

<details><summary>docker-compose logs - plugin version 6.2.7</summary>

```
postgres-db  |                                                                        ,---.
postgres-db  | ,--------.,--.                 ,--.            ,---.    ,--. ,--------.|   |
postgres-db  | '--.  .--'`--',--.--. ,---.  ,-|  |     ,---. /  .-'    |  | '--.  .--'|  .'
postgres-db  |    |  |   ,--.|  .--'| .-. :' .-. |    | .-. ||  `-,    |  |    |  |   |  |
postgres-db  |    |  |   |  ||  |   \   --.\ `-' |    ' '-' '|  .-'    |  |.--.|  |   `--'
postgres-db  |    `--'   `--'`--'    `----' `---'      `---' `--'      `--''--'`--'   .--.
postgres-db  |                                                                        '--'
postgres-db  | Image:  tiredofit/postgres:15 | Version  15-3.1.0 Type 'image_changelog' for details
postgres-db  | Repository/Documentation: https://github.com/tiredofit/docker-postgres/
postgres-db  | 
postgres-db  | If this image provides you value  - Consider sponsoring my work for continued 
postgres-db  | development, timely updates, and feature requests. Commercial support available.
postgres-db  | 
postgres-db  |                     More Info:  https://www.tiredofit.ca
postgres-db  | 
postgres-db  | 2023-05-09.07:01:40 [NOTICE] ** [container-init] Detected Container that has been restarted - Cleaning '/tmp/.container' files
postgres-db  | 2023-05-09.07:01:40 [NOTICE] ** [monitoring] Container configured for monitoring with 'zabbix modern'
postgres-db  | 2023-05-09.07:01:40 [NOTICE] ** [scheduling] Container configured for scheduled tasks with 'cron'
postgres-db  | 2023-05-09.07:01:40 [NOTICE] ** [postgres] [create_databases] Creating User: 'user'
postgres-db  | 2023-05-09.07:01:40 [NOTICE] ** [postgres] [create_databases] Granting all privileges to 'user' on 'database'
postgres-db  | 2023-05-09.07:01:40 [INFO] ** [postgres] PostgreSQL init process complete - ready for start up.
postgres-db  | 2023-05-09.07:01:40 [STARTING] ** [monitoring] [1] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:40 [STARTING] ** [postgres] [1] Starting PostgreSQL 15.2
postgres-db  | 2023-05-09.07:01:40 [STARTING] ** [scheduling] [1] Starting cron
postgres-db  | 2023-05-09.07:01:41 [STARTING] ** [monitoring] [2] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:42 [STARTING] ** [monitoring] [3] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:44 [STARTING] ** [monitoring] [4] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:45 [STARTING] ** [monitoring] [5] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:46 [STARTING] ** [monitoring] [6] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:47 [STARTING] ** [monitoring] [7] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:48 [STARTING] ** [monitoring] [8] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:49 [STARTING] ** [monitoring] [9] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:50 [STARTING] ** [monitoring] [10] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:51 [STARTING] ** [monitoring] [11] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:52 [STARTING] ** [monitoring] [12] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:54 [STARTING] ** [monitoring] [13] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:55 [STARTING] ** [monitoring] [14] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:56 [STARTING] ** [monitoring] [15] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:57 [STARTING] ** [monitoring] [16] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:58 [STARTING] ** [monitoring] [17] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:01:59 [STARTING] ** [monitoring] [18] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:00 [STARTING] ** [monitoring] [19] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:01 [STARTING] ** [monitoring] [20] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:02 [STARTING] ** [monitoring] [21] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:04 [STARTING] ** [monitoring] [22] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:05 [STARTING] ** [monitoring] [23] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:06 [STARTING] ** [monitoring] [24] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:07 [STARTING] ** [monitoring] [25] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:08 [STARTING] ** [monitoring] [26] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:09 [STARTING] ** [monitoring] [27] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:10 [STARTING] ** [monitoring] [28] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:11 [STARTING] ** [monitoring] [29] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:12 [STARTING] ** [monitoring] [30] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:14 [STARTING] ** [monitoring] [31] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:15 [STARTING] ** [monitoring] [32] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:16 [STARTING] ** [monitoring] [33] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:17 [STARTING] ** [monitoring] [34] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:18 [STARTING] ** [monitoring] [35] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:19 [STARTING] ** [monitoring] [36] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:20 [STARTING] ** [monitoring] [37] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:21 [STARTING] ** [monitoring] [38] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:22 [STARTING] ** [monitoring] [39] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:24 [STARTING] ** [monitoring] [40] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:25 [STARTING] ** [monitoring] [41] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:26 [STARTING] ** [monitoring] [42] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:27 [STARTING] ** [monitoring] [43] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:28 [STARTING] ** [monitoring] [44] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:29 [STARTING] ** [monitoring] [45] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:30 [STARTING] ** [monitoring] [46] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:31 [STARTING] ** [monitoring] [47] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:32 [STARTING] ** [monitoring] [48] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:34 [STARTING] ** [monitoring] [49] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:02:35 [STARTING] ** [monitoring] [50] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023/05/09 07:02:35.230815 cannot register plugins: plugin cannot be loaded by agent using protocol version 6.4.0, the supported version is 6.2.7
postgres-db  | zabbix_agent2 [6852]: ERROR: cannot register plugins: plugin cannot be loaded by agent using protocol version 6.4.0, the supported version is 6.2.7
postgres-db  | 2023-05-09.07:02:36 [ERROR] ** [monitoring] POTENTIAL RUNWAY DETECTECTED: Disabling 03-monitoring service because it has tried restarting '50' times
```

</details>

From within the container: 
- `POSTGRES_ZABBIX_PLUGIN_VERSION=6.2.7`
- `ZABBIX_VERSION=6.4.2`

Is this version mismatch the issue? 

When I build the image locally with the following command, it builds and runs without the runaway for the zabbix agent:

```
sudo docker build -t radarsymphony/postgres:15-6.4.2 --build-arg="POSTGRES_ZABBIX_PLUGIN_VERSION=6.4.2" .
```

<details><summary>docker-compose logs - plugin version 6.4.2</summary>

```
[+] Running 1/1
 ⠿ Container postgres-db  Recreated                                                                                                                                                        0.2s
Attaching to postgres-db
postgres-db  |                                                                        ,---.
postgres-db  | ,--------.,--.                 ,--.            ,---.    ,--. ,--------.|   |
postgres-db  | '--.  .--'`--',--.--. ,---.  ,-|  |     ,---. /  .-'    |  | '--.  .--'|  .'
postgres-db  |    |  |   ,--.|  .--'| .-. :' .-. |    | .-. ||  `-,    |  |    |  |   |  |
postgres-db  |    |  |   |  ||  |   \   --.\ `-' |    ' '-' '|  .-'    |  |.--.|  |   `--'
postgres-db  |    `--'   `--'`--'    `----' `---'      `---' `--'      `--''--'`--'   .--.
postgres-db  |                                                                        '--'
postgres-db  | Image:  tiredofit/postgres:15
postgres-db  | Repository/Documentation: https://github.com/tiredofit/docker-postgres/
postgres-db  | 
postgres-db  | If this image provides you value  - Consider sponsoring my work for continued 
postgres-db  | development, timely updates, and feature requests. Commercial support available.
postgres-db  | 
postgres-db  |                     More Info:  https://www.tiredofit.ca
postgres-db  | 
postgres-db  | 2023-05-09.14:49:23 [NOTICE] ** [timezone] Timezone: Setting to 'America/Vancouver' from 'Etc/GMT'
postgres-db  | 2023-05-09.07:49:23 [NOTICE] ** [monitoring] Container configured for monitoring with 'zabbix modern'
postgres-db  | 2023-05-09.07:49:23 [NOTICE] ** [scheduling] Container configured for scheduled tasks with 'cron'
postgres-db  | 2023-05-09.07:49:24 [NOTICE] ** [postgres] [create_databases] Creating User: 'user'
postgres-db  | 2023-05-09.07:49:24 [NOTICE] ** [postgres] [create_databases] Granting all privileges to 'user' on 'database'
postgres-db  | 2023-05-09.07:49:24 [INFO] ** [postgres] PostgreSQL init process complete - ready for start up.
postgres-db  | 2023-05-09.07:49:24 [STARTING] ** [monitoring] [1] Starting Zabbix Agent (modern) 6.4.2
postgres-db  | 2023-05-09.07:49:24 [STARTING] ** [postgres] [1] Starting PostgreSQL 15.2
postgres-db  | 2023-05-09.07:49:24 [STARTING] ** [scheduling] [1] Starting cron
```

</details>

Are you able to replicate on your end?

**Comment by tiredofit** [comment]
Fixed in `15-3.1.1`
https://github.com/tiredofit/docker-postgres/commit/f9b5da0aa6a1957d558745a20cd7dc7227e3593a

**Comment by radarsymphony** [comment]
Wicked! That has resolved the issue for me. Thank you.
